### ISSUE TITLE
[Syntax Highlight] Typ and Function argument 

### ISSUE THREAD

**Issue by senyaak** [TypeStrong/atom-typescript#1465]
If I have some Type declared in my file - and after that I use the same name as function argument the name get a wrapper with classes: syntax--support syntax--type
This brake syntax highlight.
Code to reproduce: 
```ts
class Foo {}

function (
  bar: A, 
  Foo: number, 
  fn: string,
  A: null,
)  {
  
}
```

PS. The argument names which contain only 1 character - have the same problem

![screenshot from 2018-11-15 13-48-28](https://user-images.githubusercontent.com/15360093/48553919-3df9d600-e8dd-11e8-897c-2acde85b6e69.png)

**Comment by lierdakil** [comment]
Please see: https://github.com/TypeStrong/atom-typescript/blob/master/docs/faq.md#syntax-highlighting-is-bad
TL;DR: We're not responsible for the syntax highlighting since a while ago. The relevant repo is https://github.com/atom/language-typescript
